aviary/variable_info/variable_meta_data.py

```diff
@@ -6085,6 +6212,17 @@
     desc="Atmospheric dynamic pressure at the vehicle's current flight condition"
 )
 
+add_meta_data(
+    Dynamic.Mission.ELECTRIC_ENERGY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='kJ',
+    desc='Total amount of electric energy consumed by the vehicle across the mission',
```

Comment:
is this across the whole mission, or just up until the current time?

Reply:
The latter, I've changed it to `'Total amount of electric energy consumed by the vehicle up until this point in the mission'`